sites/geohub/src/components/VectorLayer.svelte

```diff
@@ -67,7 +67,7 @@
     <Tabs
       bind:tabs
       bind:activeTab
-      fontSize="small"
+      fontSize="normal"
```

Comment:
I don't think there is `normal` value is supported by the Tabs component. Please check Tabs.svelte in UNDP design. It is only supporting either `medium` or `small`